app/API/V1/Controllers/UserController.php

```diff
@@ -2,34 +2,38 @@
 
```

Comment:
If everything is working can you remove the old commented out code? @jerazo . I had commented out just to keep it for reference, but if everything is working we don't need it any more.